kernel/src/fs/mod.rs

```diff
@@ -67,4 +68,121 @@ pub fn lazy_init() {
         println!("[kernel] Mount ExFat fs at {:?} ", target_path);
         self::rootfs::mount_fs_at(exfat_fs, &target_path).unwrap();
     }
+
+    #[cfg(feature = "raid_test")]
+    init_raid1_test_device();
+}
+
+#[cfg(feature = "raid_test")]
+const RAID_TEST_DEVICE_NAME: &str = "raid1-test";
+
+#[cfg(feature = "raid_test")]
+const RAID_TEST_MEMBER_NAMES: &[&str] = &["raid0", "raid1"];
+
+#[cfg(feature = "raid_test")]
+fn init_raid1_test_device() {
+    info!(
+        "[raid-test] initializing RAID-1 '{}' with members {:?}",
+        RAID_TEST_DEVICE_NAME, RAID_TEST_MEMBER_NAMES
+    );
+
+    let mut members = Vec::with_capacity(RAID_TEST_MEMBER_NAMES.len());
+
+    for &name in RAID_TEST_MEMBER_NAMES {
+        match start_block_device(name) {
+            Ok(device) => {
+                info!("[raid-test] member '{}' online", name);
+                members.push(device);
+            }
+            Err(err) => {
+                error!(
+                    "[raid-test] failed to start member '{}': {:?}. RAID-1 test disabled",
+                    name, err
+                );
+                return;
+            }
+        }
+    }
+
+    let raid = Raid1Device::register(RAID_TEST_DEVICE_NAME, members);
+    let worker = raid.clone();
+    let task_fn = move || loop {
+        worker.handle_requests();
+    };
+    crate::ThreadOptions::new(task_fn).spawn();
```

Comment:
This seems like something that needs to be started as part of the RAID device initialization. How do other block devices handle this? Do they also have their own threads?